sentry_docker_conf.py

```diff
@@ -98,6 +98,9 @@ def nydus_config(from_env_var):
 # You MUST configure the absolute URI root for Sentry:
 SENTRY_URL_PREFIX = config('SENTRY_URL_PREFIX')  # No trailing slash!
 
+SCRIPT_NAME = config('SCRIPT_NAME', '')
+FORCE_SCRIPT_NAME = config('FORCE_SCRIPT_NAME', '')
```

Comment:
For the sake of consistency with [Django]](https://docs.djangoproject.com/en/1.6/ref/settings/#force-script-name) the default should be ``None``.